src/proxy.rs

```diff
@@ -28,9 +35,9 @@ use {into_url, IntoUrl, Url};
 /// check each `Proxy` in the order it was added. This could mean that a
 /// `Proxy` added first with eager intercept rules, such as `Proxy::all`,
 /// would prevent a `Proxy` later in the list from ever working, so take care.
-#[derive(Clone, Debug)]
+#[derive(Debug)]
```

Comment:
Woops, I didn't notice last time, but removing the `Clone` impl would be a breaking change.

Now, that doesn't necessarily mean it absolutely must implement `Clone`. Does it make sense for `Proxy` to be clone? If it doesn't, we can leave it off, and consider that this PR must be part of 0.9...

Reply:
Added back.